README.md

```diff
@@ -6,7 +6,7 @@ Turn your browser into a Neovim client.
 
 # How to use
 
-Just click on any textarea and it will be immediately replaced by an instance of Firenvim. When you want to set the content of the now hidden textarea to the content of the Neovim instance, just `:w`. If you want to close the Firenvim overlay and return to the textarea run `:q`. If you selected an element where you expected the Firenvim frame to appear and it didn't, try pressing `<C-e>` (you can configure this binding in `about://addons` or `chome://extensions`), this should manually trigger firenvim. Please also open an issue in order to let us know the website and element this happened on.
+Just click on any textarea and it will be immediately replaced by an instance of Firenvim. When you want to set the content of the now hidden textarea to the content of the Neovim instance, just `:w`. If you want to close the Firenvim overlay and return to the textarea run `:q`. If you selected an element where you expected the Firenvim frame to appear and it didn't, try pressing `<C-e>` (you can configure this binding in [the shortcuts menu in `about:addons`](https://support.mozilla.org/en-US/kb/manage-extension-shortcuts-firefox) on Firefox, or `chome://extensions/shortcuts` on Chrome), this should manually trigger firenvim. Please also open an issue in order to let us know the website and element this happened on.
```

Comment:
Maybe we could also add here as well that the shortcut overrides the fact that firenvim is deactivated?

Reply:
Yeah, I agree with you, this point still isn't made clear from this paragraph. I think the problem stems from the terminology being used, setting `selector` to `''` doesn't actually deactivate Firenvim, it just stops it from automatically taking over elements. I think we need to find the right word to express this.